api/internal/handler/upstream/upstream.go

```diff
@@ -144,8 +146,33 @@ type BatchDelete struct {
 
 func (h *Handler) BatchDelete(c droplet.Context) (interface{}, error) {
 	input := c.Input().(*BatchDelete)
+	ids := strings.Split(input.IDs, ",")
 
-	if err := h.upstreamStore.BatchDelete(c.Context(), strings.Split(input.IDs, ",")); err != nil {
+	for _, id := range ids {
```

Comment:
why do we need those codes?

they are not E2E test codes.

Reply:
removed.